src/contracts/Repository/Collections/TotalNonPaginatedCountableInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Core\Repository\Collections;
+
+interface TotalNonPaginatedCountableInterface
```

Comment:
Descriptive, true...

But I can imagine the confusion of people seeing this interface for the first time... :sweat_smile:

Reply:
`TotalCountAwareInterface`?